templates/download/amd-xilinx/tab-3.html

```diff
@@ -0,0 +1,84 @@
+<div tabindex="0" role="tabpanel" id="versal-adaptive-evaluation-kits-tab" aria-labelledby="versal-adaptive-evaluation-kits" aria-hidden="false">
+  <div class="row u-vertically-center u-no-padding">
+    <div class="col-6">
+      <h2>Download Ubuntu Desktop</h2>
+      <h3 class="p-heading--4">Ubuntu Desktop 22.04 LTS</h3>
+      <p class="u-sv1">Ubuntu Desktop 22.04 (Developer Preview)</p>
+      <p>The version of Ubuntu is meant for evaluation and development purposes only. It should not be used in production systems.</p>
+      <div class="p-notification--information is-inline">
+        <div class="p-notification__content">
+          <h5 class="p-notification__title">Note:</h5>
+          <p class="p-notification__message">This release only supports VCK190 Evaluation Kits (non-ES).</p>
+        </div>
+      </div>
+      <p><a class="p-button--positive" href="https://people.canonical.com/~platform/images/xilinx/versal-ubuntu-22.04/iot-limerick-versal-classic-server-2204-x02-20230315-48.img.xz">Download 64-bit</a></p>
+      <p>Works on:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">AMD-Xilinx VCK190 Evaluation Kits (non-ES)</li>
+      </ul>
+      <p>Known limitations:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">System exhibits instability issues upon resuming from suspend mode</li>
+        <li class="p-list__item is-ticked">For a full list of known limitations, please visit <a href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/1413611532/Canonical+Ubuntu">the wiki</a></li>
+      </ul>
+    </div>
+    <div class="col-6 u-hide--medium u-hide--small u-align--center">
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/18d7b3ce-AMD-Xilinx-VCK190.png",
+        alt="AMD-Xilinx VCK190 circuit board",
+        width="503",
+        height="283",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <h4>First time installing Ubuntu on AMD-Xilinx?</h4>
+    <p>Please see the <a href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/2363129857/Getting+Started+with+Certified+Ubuntu+22.04+LTS+for+Xilinx+Devices">Getting Started with Certified Ubuntu 22.04 LTS for AMD-Xilinx Devices</a> for more information.</p>
+    <p><strong>AMD-Xilinx Ubuntu Software Development Kit</strong></p>
+    <p>Ubuntu developers targeting the Versal&trade; Evaluation Kits may also want to download the following files:</p>
+    
+    <table aria-label="AMD-Xilinx Ubuntu Software Development Kit table">
+      <thead>
+        <tr>
+          <th>Image name</th>
+          <th colspan="2">Description</th>
```

Comment:
`colspan` is to merge cells and in this case, there are no cells to merge into causing invalid HTML. I would suggest removing the col spans as it looks nice without them. If you want a different width you can add `width="65%"` to the table header and all child cells in that column will inherit it.

Reply:
The width doesn't look great on small screens: 
![image](https://github.com/canonical/ubuntu.com/assets/1413534/35a0e67d-f571-46aa-89e8-cfe5d21528d9)